docs/source/settings.rst

```diff
@@ -310,7 +312,7 @@ chdir
 ~~~~~
 
 * ``--chdir``
-* ``/Users/benoitc/work/gunicorn/py27/gunicorn/docs``
+* ``/Users/slava/Desktop/gunicorn/docs``
```

Comment:
This looks like an unrelated change(and caused by `make -C docs html`) to me. Can you revert it? :)